tests/conftest.py

```diff
@@ -69,14 +69,18 @@ def pytest_configure(config):
         cloud_keyword = cloud_to_pytest_keyword[cloud]
         config.addinivalue_line(
             'markers', f'{cloud_keyword}: mark test as {cloud} specific')
+
```

Comment:
Unintended?

Reply:
This was a fix to the previous unintended new line https://github.com/skypilot-org/skypilot/pull/1736#discussion_r1143984452